pkg/kubectl/cmd/expose.go

```diff
@@ -152,7 +152,7 @@ func NewCmdExposeService(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	cmd.Flags().String("container-port", "", i18n.T("Synonym for --target-port"))
 	cmd.Flags().MarkDeprecated("container-port", "--container-port will be removed in the future, please use --target-port instead")
 	cmd.Flags().String("target-port", "", i18n.T("Name or number for the port on the container that the service should direct traffic to. Optional."))
-	cmd.Flags().String("external-ip", "", i18n.T("Additional external IP address (not managed by Kubernetes) to accept for the service. If this IP is routed to a node, the service can be accessed by this IP in addition to its generated service IP."))
+	cmd.Flags().String("external-ips", "", i18n.T("Additional external IP addresses (not managed by Kubernetes) to accept for the service. If these IPs are routed to a node, the service can be accessed by these IPs in addition to its generated service IP."))
```

Comment:
Why not setting this as `StringSlice` or `StringArray`?

Reply:
Good idea! I will change this.